### PR TITLE
Use actions/checkout@v3

### LINKED ISSUE #61: Frontend GHA warning in test job about Node.js 12
Node.js 12 actions are deprecated. Please update the following actions to use Node.js 16: actions/checkout@v2. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/.

### PR DESCRIPTION
**Pull request by Fosten**
Fixes #61

### CODE REVIEW

**Review by stevepiercy — APPROVED** [review]
Thank you! Let's get a maintainer's approval, too.

**Review by jensens — APPROVED** [review]

**Pull request merged**
